src/outofprocerrorwrapper.ts

```diff
@@ -0,0 +1,11 @@
+import { OrchestratorState } from "./classes";
+
+const outOfProcDataLabel = "\n\n$OutOfProcData$:";
+
+export class OutOfProcErrorWrapper extends Error {
+    constructor(error: Error, state: OrchestratorState) {
+        const message = `${error.message}${outOfProcDataLabel}${JSON.stringify(state)}`;
```

Comment:
I suggest you add a comment explaining that this is an predefined contract understood by the Durable extension, and that changing this could be a breaking change.

Reply:
Adding this as a class level comment.